integration-tests/src/tests/client/features/delegate_action.rs

```diff
@@ -572,3 +576,148 @@ fn assert_ft_balance(
     let balance = std::str::from_utf8(&response.result).expect("invalid UTF8");
     assert_eq!(format!("\"{expected_balance}\""), balance);
 }
+
+/// Test account creation scenarios with meta transactions.
+///
+/// Named accounts aren't the primary use case for meta transactions but still
+/// worth a test case.
+#[test]
+fn meta_tx_create_named_account() {
+    let relayer = bob_account();
+    let sender = alice_account();
+    let new_account = eve_dot_alice_account();
+    let node = RuntimeNode::new(&relayer);
+
+    let fee_helper = fee_helper(&node);
+    let amount = NEAR_BASE;
+
+    let public_key = PublicKey::from_seed(KeyType::ED25519, &new_account);
+
+    // That's the minium to create a (useful) account.
```

Comment:
```suggestion
    // That's the minimum to create a (useful) account.
```
nit :) and I would make sentences below more complete

Reply:
done